hogben/simulate.py

```diff
@@ -26,51 +51,48 @@ def simulate_magnetic(sample, angle_times, scale=1, bkg=5e-7, dq=2,
         mp (bool): whether to simulate "minus plus" spin state.
         mm (bool): whether to simulate "minus minus" spin state.
         angle_scale (float): angle to use when scaling directbeam flux.
-        directbeam_path (str): path to directbeam file to simulate with.
+        instrument (str): path to directbeam file to simulate with.
 
     Returns:
         tuple: model and simulated data for the given `sample`.
 
     """
     models, datasets = [], []
 
-    # Default path to the directbeam file for OFFSPEC when polarised.
-    if directbeam_path is None:
-        directbeam_path = importlib_resources.files('hogben.data.directbeams').joinpath('OFFSPEC_polarised_old.dat')
-        angle_scale = 0.3
-
+    instrument_DB = DB_path(instrument, polarised=True)
```

Comment:
Variable naming, to an idiot (me) looking at this `instrument_DB` looks like a database of instruments (wait, maybe it is?)

Also the use of capitals in a variable name is a bit jarring, but maybe this is already done elsewhere in HOGBEN and fits with the style?